perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp

```diff
@@ -307,6 +280,99 @@ TrackedObjects RadarTracksMsgsConverterNode::convertRadarTrackToTrackedObjects()
   return tracked_objects;
 }
 
+geometry_msgs::msg::Vector3 RadarTracksMsgsConverterNode::compensateVelocity(
```

Comment:
[IMO]
void compensateEgoMotion() is the same comment as void convertCovarianceMatrix()

Reply:
as same as above, the function returns value, and cases an additional copy operation.

Reply:
fixed at [7a35a7f](https://github.com/autowarefoundation/autoware.universe/pull/5823/commits/7a35a7f10b2ddff48fb259561afdb3483e40e7c5)